cmd/git-sync/main.go

```diff
@@ -78,8 +82,30 @@ var flCookieFile = flag.Bool("cookie-file", envBool("GIT_COOKIE_FILE", false),
 var flGitCmd = flag.String("git", envString("GIT_SYNC_GIT", "git"),
 	"the git command to run (subject to PATH search)")
 
+var flHTTPBind = flag.String("http-bind", envString("GIT_SYNC_HTTP_BIND", ""),
+	"the bind address for git-sync's HTTP endpoint")
```

Comment:
nit: `"the bind address including port for git-sync's HTTP endpoint"`
It's assumed, but making it explicit might reduce additional questions.